frontend/src/components/Step5.tsx

```diff
@@ -0,0 +1,66 @@
+import { Button } from "./Button.tsx";
+import styles from "./Step5.module.css";
+
+// Pass in Application's next function
+export type StepProps = {
+  next: () => void;
+};
+
+export const Step5: React.FC<StepProps> = ({ next }: StepProps) => {
+  // Checks if all Required Inputs are filled before moving on to next step
```

Comment:
I'm a little confused as to why Step5 is a form. Also, why we are checking that all required inputs are filled before moving on? From what I can tell the step has no inputs. Thanks